public/app/features/alerting/unified/components/expressions/util.test.ts

```diff
@@ -0,0 +1,80 @@
+import { DataFrame, FieldType, toDataFrame } from '@grafana/data';
+
+import { getSeriesName, formatLabels, getSeriesValue, isEmptySeries } from './util';
+
+const EMPTY_FRAME: DataFrame = toDataFrame([]);
+const NAMED_FRAME: DataFrame = {
+  name: 'MyFrame',
+  ...toDataFrame([]),
+};
+
+const DATA_FRAME: DataFrame = toDataFrame({
+  fields: [{ name: 'value', type: FieldType.number, values: [1, 2, 3] }],
+});
+
+const DATA_FRAME_LARGE_DECIMAL: DataFrame = toDataFrame({
+  fields: [{ name: 'value', type: FieldType.number, values: [1.23456789] }],
+});
+
+const DATA_FRAME_WITH_LABELS: DataFrame = toDataFrame({
+  fields: [{ name: 'value', type: FieldType.number, values: [1, 2, 3], labels: { foo: 'bar' } }],
+});
+
+describe('formatLabels', () => {
+  it('should work with no labels', () => {
+    expect(formatLabels({})).toBe('');
+  });
+
+  it('should work with 1 label', () => {
+    expect(formatLabels({ foo: 'bar' })).toBe('foo=bar');
+  });
+
+  it('should work with multiple labels', () => {
+    expect(formatLabels({ foo: 'bar', baz: 'qux' })).toBe('foo=bar, baz=qux');
```

Comment:
What should happen if the label value contains whitespaces or commas?

Reply:
I'm currently not trimming any whitespaces, if spaces (or commas) are in the label values it's probably because they're allowed by the data source. The presentation _would_ look rather weird admittedly.